### DISCUSSION TITLE
以下的代码为什么只触发了onMouseDown事件但并不触发onClick

### DISCUSSION THREAD

**Discussion by Sharang-heng** [ant-design/ant-design#40522] [Q&A]
` <div`
`    onClick={() => {`
`        console.log('onClick');`
 `         onInputEnter(inputValue);`
`     }}`
              `     onMouseDown={() => {`
                `        console.log('onMouseDown');`
                `        onInputEnter(inputValue);`
             `     }}`
            `    >`
             `     <Icon`
               `     style={{ color: '#8F9399', width: 16, height: 16, zIndex: 100 }}`
               `     component={() => (`
                  `        <svg xmlns="https://www.w3.org/2000/svg" fill="none" viewBox="0 0 48 48">`
                    `        <path`
                      `        strokeLinejoin="round"`
                      `        strokeWidth="4"`
                     `         stroke="#8F9399"`
                    `         d="M21 38c9.389 0 17-7.611 17-17S30.389 4 21 4 4 11.611 4 21s7.611 17 17 17Z"`
                      `        data-follow-stroke="#333"`
                    `        />`
                    `        <path`
                      `        strokeLinejoin="round"`
                      `        strokeLinecap="round"`
                      `        strokeWidth="4"`
                      `        stroke="#8F9399"`
                      `        d="M26.657 14.343A7.975 7.975 0 0 0 21 12c-2.209 0-4.209.895-5.657 2.343M33.222 33.222l8.485 8.485"`
                      `        data-follow-stroke="#333"`
                    `    />`
                  `    </svg>`
               `     )}`
              `    />`
            `</div>`

**Comment by zombieJ** [comment]
没有
